src/redisraft.h

```diff
@@ -519,6 +523,7 @@ typedef struct {
     bool asking;                  /* if this command array is in an asking mode */
     int size;                     /* Size of allocated array */
     int len;                      /* Number of elements in array */
+    size_t cmd_flags;             /* the calculated cmd_flags for all commands in this array */
```

Comment:
I think `size_t` gives wrong impression here for flags (feels like it is the length of an array, number of items etc.).
maybe we can use `unsigned long` or `unsigned int`.

Reply:
ok